### PR TITLE
Add pipelines for `@pagopa/express-azure-functions`

### PR DESCRIPTION
**Pull request by gquadrati**
Add  `@pagopa/express-azure-functions` pipelines to `pagopa-packages-projects`

### CODE REVIEW

**Review by pasqualedevita — CHANGES_REQUESTED** [review]
express-azure-functions is not found... maybe io-functions-express?
https://github.com/pagopa/express-azure-functions -> 404

please use terraform module like in https://github.com/pagopa/gitops/blob/main/azure-devops/projects/eucovidcert-projects/io-functions-eucovidcerts.tf

**Comment by gquadrati** [comment]
> express-azure-functions is not found... maybe io-functions-express?
> https://github.com/pagopa/express-azure-functions -> 404
> 
> please use terraform module like in https://github.com/pagopa/gitops/blob/main/azure-devops/projects/eucovidcert-projects/io-functions-eucovidcerts.tf

thanks, done in [b0f3d4d](https://github.com/pagopa/gitops/commit/b0f3d4d4f2a8b2af6528d0958ebf242d565da4c8)

File: azure-devops/projects/pagopa-packages-projects/express-azure-functions.tf [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+  }
+  # deploy vars
+  express-azure-functions-variables_deploy = {
+    git_mail                       = module.secrets.values["io-azure-devops-github-EMAIL"].value
+    git_username                   = module.secrets.values["io-azure-devops-github-USERNAME"].value
+    github_connection              = azuredevops_serviceendpoint_github.io-azure-devops-github-rw.service_endpoint_name
+    npm_connection                 = azuredevops_serviceendpoint_npm.pagopa-npm-bot.service_endpoint_name
+    production_resource_group_name = "io-p-rg-eucovidcert"
+    production_app_name            = "io-p-fn3-eucovidcert"
+    production_azure_subscription  = azuredevops_serviceendpoint_azurerm.PROD-IO.service_endpoint_name
```
Comment (pasqualedevita) [inline azure-devops/projects/pagopa-packages-projects/express-azure-functions.tf:42]:
```suggestion
```

File: azure-devops/projects/pagopa-packages-projects/express-azure-functions.tf [hunk 2]
```diff
@@ -0,0 +1,96 @@
+variable "express-azure-functions" {
```
Comment (pasqualedevita) [inline azure-devops/projects/pagopa-packages-projects/express-azure-functions.tf:1]:
```suggestion
variable "io-functions-express" {
```
change also refs

File: azure-devops/projects/pagopa-packages-projects/express-azure-functions.tf [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+  )
+
+  variables_secret = merge(
+    local.express-azure-functions-variables_secret,
+    local.express-azure-functions-variables_secret_deploy,
+  )
+
+  service_connection_ids_authorization = [
+    azuredevops_serviceendpoint_github.io-azure-devops-github-ro.id,
+    azuredevops_serviceendpoint_azurerm.PROD-IO.id,
```
Comment (pasqualedevita) [inline azure-devops/projects/pagopa-packages-projects/express-azure-functions.tf:93]:
```suggestion
```

File: azure-devops/projects/pagopa-packages-projects/express-azure-functions.tf [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+  express-azure-functions-variables_secret = {
+
+  }
+  # code_review vars
+  express-azure-functions-variables_code_review = {
+
+  }
+  # code_review secrets
+  express-azure-functions-variables_secret_code_review = {
+    danger_github_api_token = module.secrets.values["DANGER-GITHUB-API-TOKEN"].value
```
Comment (pasqualedevita) [inline azure-devops/projects/pagopa-packages-projects/express-azure-functions.tf:32]: do u use DANGER?

**Review by pasqualedevita — CHANGES_REQUESTED** [review]

File: azure-devops/projects/pagopa-packages-projects/express-azure-functions.tf (hunk 2 above)
Comment (gquadrati) [reply azure-devops/projects/pagopa-packages-projects/express-azure-functions.tf:1]: Repository is called `io-functions-express`, but the package name is now `@pagopa/azure-functions-express`.
I thought to use the same name, where we have this opportunity, leaving the old one for  repository name only.
Can we do it? 
cc: @balanza

File: azure-devops/projects/pagopa-packages-projects/express-azure-functions.tf (hunk 4 above)
Comment (gquadrati) [reply azure-devops/projects/pagopa-packages-projects/express-azure-functions.tf:32]: yes, but thanks for pointing me this condition to check

File: azure-devops/projects/pagopa-packages-projects/express-azure-functions.tf (hunk 2 above)
Comment (pasqualedevita) [reply azure-devops/projects/pagopa-packages-projects/express-azure-functions.tf:1]: variables and names on gitops refer to github repository.
We can set only repository.name = "io-functions-express" but maybe it's better to rename https://github.com/pagopa/io-functions-express

Comment (gquadrati) [reply azure-devops/projects/pagopa-packages-projects/express-azure-functions.tf:1]: I did your first option, but the second one is the best one, I agree. I don't think I can do it.

Comment (pasqualedevita) [reply azure-devops/projects/pagopa-packages-projects/express-azure-functions.tf:1]: Help @balanza :) Can u rename `io-functions-express` in `azure-functions-express`?

Comment (balanza) [reply azure-devops/projects/pagopa-packages-projects/express-azure-functions.tf:1]: done

File: azure-devops/projects/pagopa-packages-projects/express-azure-functions.tf [hunk 5]
```diff
@@ -0,0 +1,92 @@
+variable "express-azure-functions" {
+  default = {
+    repository = {
+      organization    = "pagopa"
+      name            = "io-functions-express"
```
Comment (pasqualedevita) [inline azure-devops/projects/pagopa-packages-projects/express-azure-functions.tf:5]:
```suggestion
      name            = "azure-functions-express"
```

**Review by pasqualedevita — CHANGES_REQUESTED** [review]

File: azure-devops/projects/pagopa-packages-projects/express-azure-functions.tf (hunk 5 above)
Comment (gquadrati) [reply azure-devops/projects/pagopa-packages-projects/express-azure-functions.tf]: @pasqualedevita actually npm module is named [express-azure-functions](https://www.npmjs.com/package/@pagopa/express-azure-functions) not `azure-functions-express`.
If we want to maintain coherence between github and npm, we need to rename github repo.
cc @balanza

Comment (pasqualedevita) [reply azure-devops/projects/pagopa-packages-projects/express-azure-functions.tf]: yes absolutely my mistake

Comment (gquadrati) [reply azure-devops/projects/pagopa-packages-projects/express-azure-functions.tf]: no problem. renamed in [d83c497](https://github.com/pagopa/gitops/pull/109/commits/d83c497a9f54f080e863a00c95200e6765cfc36d) :)

**Comment by pasqualedevita** [comment]
@gquadrati I updated secrets management. It's only a code styling, no changes in azure pipelines variables

**Review by pasqualedevita — APPROVED** [review]

**Pull request merged**